src/mpid/common/shm/mpidu_init_shm.c

```diff
@@ -98,7 +98,7 @@ int MPIDU_Init_shm_get(int local_rank, void **target)
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDU_INIT_SHM_SEG_GET);
 
     MPIR_Assert(local_rank < local_size);
-    *target = baseaddr + local_rank * sizeof(MPIDU_Init_shm_block_t);
+    *target = (char *) baseaddr + local_rank * sizeof(MPIDU_Init_shm_block_t);
```

Comment:
@hzhou sorry about that I should have tested the warnings before merging.

Reply:
We are all guilty of the warnings :)